Client/src/Components/NavigationMenu.tsx

```diff
@@ -1,46 +1,20 @@
 /* eslint-disable react/display-name */
-/* eslint-disable react/prop-types */
 
-import { AppBar, Box, Button, Divider, Drawer, Grid, IconButton, List, ListItem, ListItemIcon, Toolbar, Typography, makeStyles } from "@material-ui/core"
-import {
-  HashRouter,
-  Link,
-  NavLink,
-  Route
-} from "react-router-dom"
+import { AppBar, Box, Button, Divider, Drawer, Grid, IconButton, Toolbar, Typography, makeStyles } from "@material-ui/core"
+import { HashRouter, Link } from 'react-router-dom'
+import { ListRouter, getRoutedViews } from "./ListRouter"
```

Comment:
Good code quality here. Everything is listed in one line as opposed to the previous lines.